contract-variable-monitor/SETUP.md

```diff
@@ -0,0 +1,199 @@
+# Contract Variable Monitor Agent Template
+
+This agent monitors contract variable values for specified contract addresses. Alert type and severity
+are specified per variable per contract address.
+
+## Agent Setup Walkthrough
+
+The following steps will take you from a completely blank template to a functional agent.
+
+1. Open the `agent-config.json` file.
```

Comment:
I like the idea of committing `agent-config.json.example` to the template repository, then instructing the developer to copy it to `agent-config.json` when they populate it.